### PR TITLE
LPS-143943 Account Group name's validation for character limit is not consistent with other components

### PR DESCRIPTION
**Pull request by RafaelCRC**
update for [LPS-143943](https://issues.liferay.com/browse/LPS-143943)

### CODE REVIEW

**Review by gislayne-vitorino — APPROVED** [review]
LGTM! Any thoughts about it @drewbrokke ?

File: modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_groups_admin/account_group/details.jsp [hunk 1]
```diff
@@ -43,9 +43,7 @@ renderResponse.setTitle((accountGroupDisplay.getAccountGroupId() == 0) ? Languag
 			<%= LanguageUtil.get(request, "information") %>
 		</h2>
 
-		<aui:input label="account-group-name" name="name" required="<%= true %>" type="text" value="<%= accountGroupDisplay.getName() %>">
-			<aui:validator name="maxLength"><%= ModelHintsUtil.getMaxLength(AccountGroup.class.getName(), "name") %></aui:validator>
-		</aui:input>
+		<aui:input bean="<%= accountGroupDisplay.getAccountGroup() %>" label="account-group-name" model="<%= AccountGroup.class %>" name="name" />
```
Comment (drewbrokke) [inline modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_groups_admin/account_group/details.jsp:46]: Is this field still marked as `required` after this change?

Comment (RafaelCRC) [reply modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_groups_admin/account_group/details.jsp:46]: missed that, already sent the fix. Thanks!

**Comment by RafaelCRC** [comment]
ci:test:sf

**Comment by RafaelCRC** [comment]
ci:test:relevant

**Review by drewbrokke — APPROVED** [review]

**Comment by drewbrokke** [comment]
ci:forward

**Pull request closed without merging**